guibot/controller.py

```diff
@@ -167,6 +167,10 @@ def __configure_backend(
         log.log(9, "Setting backend for %s to %s", category, backend)
         self.params[category] = {}
         self.params[category]["backend"] = backend
+        self.params[category]["toggle_delay"] = 0.05
+        self.params[category]["click_delay"] = 0.1
+        self.params[category]["type_delay"] = 0.1
+        self.params[category]["keys_delay"] = 0.2
```

Comment:
Maybe we can give all of these clearer names so that they are not confused? For instance one could also have key toggle timeout which could be confused with the `toggle_delay`. Same for `keys_delay` and `type_delay` as it is not clear which delay is applied when. So let's just use:

- mouse_toggle_delay for toggle_delay
- after_click_delay for click_delay
- delay_between_keys for type_delay
- delay_before_keys for keys_delay